src/layouts/Header.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from "react-router-dom";
+
+import ActiveAlarmIcon from "../assets/icon/active.svg";
+import InActiveAlarmIcon from "../assets/icon/inactive.svg";
+import SearchIcon from "../assets/icon/search.svg";
+import Logo from "../assets/logo/thejulge.svg";
+interface HeaderProps {
+  isLoggedIn: boolean;
+  userNavLabel?: string; // "내 가게" or "내 프로필"
```

Comment:
```suggestion
  userNavLabel?: "내 가게" | "내 프로필";
```
두 가지 값에 대한 데이터만 필요하다면
리터럴 타입으로 변경해도 좋을 것 같아요. 👍